src/content_sao.cpp

```diff
@@ -156,6 +156,13 @@ LuaEntitySAO::~LuaEntitySAO()
 	if(m_registered){
 		m_env->getScriptIface()->luaentity_Remove(m_id);
 	}
+
+	std::set<u32>::iterator it;
+	std::set<u32>::iterator begin = m_attached_particle_spawners.begin();
```

Comment:
I think in most of the code you'll find that begin/end are just omitted and `m_attached_particle_spawners.begin()` is just used directly in the `for(` loop.